core/helpers/index.ts

```diff
@@ -171,3 +171,16 @@ export const processURLAddress = (url:string = '') => {
   if (url.startsWith('/')) return `${config.api.url}${url}`
   return url
 }
+
+
+/*
+  * serial executes Promises sequentially.
+  * @param {funcs} An array of funcs that return promises.
+  * @example
+  * const urls = ['/url1', '/url2', '/url3']
+  * serial(urls.map(url => () => $.ajax(url)))
+  *     .then(Logger.log.bind(Logger))()
+  */
+ export const serial = funcs =>
```

Comment:
i believe we shouldn't have that kind of helper, synchronous call of promisses array (especially when we don't care about results) could be done just with `for` loop
`for (const item in promissesArray) await item`

i've prepared jsFiddle with example when we want to do something with results (in this case just collect them to ensure that they were revolved correctly)
https://jsfiddle.net/j8eq6tfv/1/

Reply:
OK, I kept this helper but refactored it to the async manner